packages/react/src/components/Popup/Popup.tsx

```diff
@@ -497,4 +501,14 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
       onClose && onClose()
     }
   }
+
+  /**
+   * Save DOM element which had focus before Popup opens.
+   * Can be either trigger DOM element itself or the element inside it.
+   */
+  private updateTriggerFocusableDomElement() {
```

Comment:
Do we need to introduce a new method for a single use?

Reply:
I think it makes sense, don't want to mess up `tryOpen` method and still remain better reading